department-of-veterans-affairs/va.gov-cms
# VAGOV-2287: Menu export config and data for main nav

To test:
1. Go to /admin/config/development/menu_export/import. "Menus to Import" should list "Main navigation" only.
2. Click "Import Menu Links". Menu should import successfully.
3. Go to main navigation menu (/admin/structure/menu/manage/main) and make sure the menu structure matches the megamenu on va.gov.
